test/accessibility/apps/dashboard.ts

```diff
@@ -20,27 +20,36 @@
 import { FtrProviderContext } from '../ftr_provider_context';
 
 export default function({ getService, getPageObjects }: FtrProviderContext) {
-  const PageObjects = getPageObjects(['common', 'dashboard', 'header']);
+  const PageObjects = getPageObjects(['common', 'dashboard', 'header', 'home', 'settings']);
   const a11y = getService('a11y');
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
+  const dashboardPanelActions = getService('dashboardPanelActions');
+  const dashboardAddPanel = getService('dashboardAddPanel');
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const listingTable = getService('listingTable');
 
   describe('Dashboard', () => {
     const dashboardName = 'Dashboard Listing A11y';
+    const clonedDashboardName = 'Dashboard Listing A11y Copy';
+
     before(async () => {
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await kibanaServer.uiSettings.update({
-        defaultIndex: 'logstash-*',
-      });
-      await PageObjects.common.navigateToApp('dashboard');
+      // await esArchiver.loadIfNeeded('logstash_functional');
```

Comment:
Why is this commented out?

Reply:
Earlier round of phase 1 tests used logstash data. I removed it and using sample data. Removed the commented out lines. Thanks @majagrubic